src/makeCalculator.js

```diff
@@ -4,7 +4,30 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
-}
+  return {
+    result: 0,
+    add(a, b) {
+      return a + b;
+    },
+    subtract(a, b) {
+      return a - b;
+    },
+    multiply(a, b) {
+      return a * b;
```

Comment:
[CODE STYLE] - For the `multiply` method, use the `*=` operator to update the `result` property.